plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -335,4 +343,108 @@ protected TestTable createTableWithUnsupportedColumn()
 
         return Optional.of(dataMappingTestSetup);
     }
+
+    @Test
+    public void testAggregationPushdown()
+            throws Exception
+    {
+        // TODO support aggregation pushdown with GROUPING SETS
+        // TODO support aggregation over expressions
+
+        assertThat(query("SELECT count(*) FROM nation")).isFullyPushedDown();
```

Comment:
Can you please copy this again from the PostgreSQL tests (TestPostgresqlConnectorTest#testAggregationPushdown)? Some new tests were added there.

We plan to extract this test into a common superclass (BaseConnectorTest/BaseJdbcConnectorTest) so that it's easier to keep the tests in sync.

Note that instead of removing the assertions that fail (`GROUP BY over TOPN` and `GROUP BY over JOIN` are expected to fail) change the assertions so that we can verify a particular feature is not implemented.

Reply:
Same for the `testLimitPushdown` method.